### PR TITLE
Add ironic podified job to test IPA image

### PR DESCRIPTION
**Pull request by raukadah**
Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1939

### CODE REVIEW

**Comment by raukadah** [comment]
Based on [eib-podified-multinode-ironic-deployment](https://review.rdoproject.org/zuul/build/35cbe6adb78a4b9aa94d48f8bc57557f) job results.

[Data coming from edpm image build content provider](https://logserver.rdoproject.org/34/34/57a8c11b9d361d2e6a6874cc84e8c05c7be977cd/github-check/eib-podified-multinode-ironic-deployment/35cbe6a/zuul-info/inventory.yaml)
```
cifmw_build_images_output:
        images:
          edpm-hardened-uefi:
            image: 38.102.83.158:5001/edpm-hardened-uefi:57a8c11b9d361d2e6a6874cc84e8c05c7be977cd
          ironic-python-agent:
            image: 38.102.83.158:5001/ironic-python-agent:57a8c11b9d361d2e6a6874cc84e8c05c7be977cd
```
[Update containers](https://logserver.rdoproject.org/34/34/57a8c11b9d361d2e6a6874cc84e8c05c7be977cd/github-check/eib-podified-multinode-ironic-deployment/35cbe6a/controller/ci-framework-data/artifacts/manifests/update_containers.yml) updating the image
```
apiVersion: core.openstack.org/v1beta1
kind: OpenStackVersion
metadata:
  name: openstack-galera-network-isolation
  namespace: openstack
spec:
  customContainerImages:
    osContainerImage: 38.102.83.158:5001/edpm-hardened-uefi:57a8c11b9d361d2e6a6874cc84e8c05c7be977cd
    ironicPythonAgentImage: 38.102.83.158:5001/ironic-python-agent:57a8c11b9d361d2e6a6874cc84e8c05c7be977cd
```
[The updated IPA image](https://logserver.rdoproject.org/34/34/57a8c11b9d361d2e6a6874cc84e8c05c7be977cd/github-check/eib-podified-multinode-ironic-deployment/35cbe6a/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/ironic-conductor-0-describe)
```
  ironic-python-agent-init:
    Container ID:   cri-o://4eb02b4ad940c8cd1401be1918def9f8a11e7711b6bde20e1a71fa33131bf924
    Image:          38.102.83.158:5001/ironic-python-agent:57a8c11b9d361d2e6a6874cc84e8c05c7be977cd
    Image ID:       38.102.83.158:5001/ironic-python-agent@sha256:9622a7739fdb18a440ab168f9f070ef8df0700d83fd1b5854600256b5e5a568f
```

**Review by hjensas — APPROVED** [review]

**Review by hjensas — COMMENTED** [review]
/lgtm

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -7,3 +7,11 @@
...
     name: eib-crc-podified-edpm-baremetal
     parent: cifmw-crc-podified-edpm-baremetal
+    dependencies: ["eib-content-provider-build-images"]
+
+- job:
+    name: eib-podified-multinode-ironic-deployment
+    parent: podified-multinode-ironic-deployment
+    dependencies: ["eib-content-provider-build-images"]
+    vars:
+      cifmw_update_containers_ipa_image_url: "{{ cifmw_build_images_output['images']['ironic-python-agent']['image'] }}"
```
Comment (steveb) [inline zuul.d/jobs.yaml:17]: Does podified-multinode-ironic-deployment deploy any edpm nodes? If so we should use the built edpm image too

**Comment by raukadah** [comment]
recheck

File: zuul.d/jobs.yaml (hunk 1 above)
Comment (raukadah) [reply zuul.d/jobs.yaml:17]: @steveb Thank you for reviewing the pr. eib-crc-podified-edpm-baremetal is deploying edpm node using built uefi image. podified-multinode-ironic-deployment  will use the built ipa image. these two jobs will test both edpm images. I hope it answers the above question.

**Review by steveb — APPROVED** [review]
/lgtm

**Pull request merged**